File: trial/app/src/main/java/com/example/trial/SuggestedTunes.java
package com.example.trial;

import com.google.gson.annotations.SerializedName;

public class SuggestedTunes {

    @SerializedName("id")
    public int Id;
    @SerializedName("count")
    public int Count;
    @SerializedName("title")
    public String Title;
    @SerializedName("artist")
    public String Artist;
    @SerializedName("genre")
    public String Genre;

    public String toString() {
        return Artist + " " + Title + " " + Count + " " + Genre;
    }

}

File: TunesAPI/TunesAPI/SuggestedTunes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TunesAPI
{
    public class SuggestedTunes
    {
        public int Id { get; set; }
        public int Count { get; set; }
        public String Title { get; set; }
        public String Artist { get; set; }
        public String Genre { get; set; }
    }
}
